ajax/libs/input-autogrow/package.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "input-autogrow",
+  "filename": "input-autogrow.min.js",
+  "version": "1.1.0",
+  "description": "Autogrow plugin for inputs",
+  "license": "MIT",
+  "keywords": [
+    "autogrow",
+    "auto-grow",
+    "input",
+    "jquery",
+    "zepto",
+    "javascript",
+    "js"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "git://github.com/westonganger/input-autogrow.git"
+  },
+  "author": {
+    "name": "Weston Ganger",
+    "url": "https://westonganger.com/"
+  },
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/westonganger/input-autogrow.git",
+    "fileMap": [
+      {
+        "basePath": "dist",
+        "files": [
+          "**/*"
+        ]
+      }
```

Comment:
Let's also include the source file with the config
```js
      {
        "basePath": "src",
        "files": [
          "input-autogrow*"
        ]
      }
```

thanks.

Reply:
@extend1994 mmm, ok, can you do this? Getting my fork in sync with the upstream master is a big challenge with this big repository. Takes hours for me to perform such a small change...